src/App/KickOff.php

```diff
@@ -60,8 +74,9 @@ protected function getSections() {
     /**
      * @param Section $section
      */
-    protected function handleSection(Section $section) {
-        $url = $section->getTargetUrl();
+    protected function handleSection(Section $section, $targetUrl = null) {
```

Comment:
Passing $targetUrl here is a quick way to have something working, maybe there are better places to put that, like Configuration, what do you think?